### DISCUSSION TITLE
Support PHP language

### DISCUSSION THREAD

**Discussion by robertdrakedennis** [zed-industries/zed#6645] [Feature Requests]
Most PHP developers use Phpstorm or VScode, would be awesome to support this language 🙏

**Comment by dinhquochan** [comment]
Yes. Please 😱

**Comment by hientt53** [comment]
Yes. Please

**Comment by vigstudio** [comment]
All my team members will give up vscode to use zed if it supports php

**Comment by dinhquochan** [comment]
We've already well LSP here: [intelephense](https://github.com/bmewburn/intelephense-docs).

**Reply by JFOC** [reply]
it work with INTELEPHENSE?

**Reply by dinhquochan** [reply]
Not yet.

**Comment by jmfederico** [comment]
Supported since 0.110.2
